java/AccelerometerParser.java

```diff
@@ -52,8 +52,10 @@ public static void main(String[] args) {
     	DF3.setRoundingMode(RoundingMode.HALF_UP); // To match their implementation
 
     	// epochWriter parameters
-    	int epochPeriod = 30;
-    	DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSS");
+        int epochPeriod = 30;
+        // output time format, e.g. 2020-06-14 19:01:15.123000+0100 [Europe/London]
+        // this should be consistent with the date_parser used later in the Python code
+        final DateTimeFormatter timeFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSxxxx '['VV']'");
```

Comment:
Time format to use in the csv output files from the Java pipeline. As comment says, `2020-06-14 19:01:15.123000+0100 [Europe/London]`
This needs to match the `date_parser` function used later in the Python pipeline.